packages/__tests__/src/rounding.ts

```diff
@@ -33,3 +33,8 @@ export function round(thing: any, z = 8): any {
     };
     return recurse(thing);
 }
+
+export function roundString(str: string, z = 8): string {
+    const r = makeRound(z);
+    return str.replace(/\d+\.\d+/g, (n) => r(parseFloat(n)).toString());
```

Comment:
regexp for number matching is hard. eg `-` vs `0x10` vs `1e10` `5n`

would be safer to split on a known eg `foo.split(` `).map(r).join(' ')`

Reply:
This is just for testing with numbers with an explicit decimal point in it. We won't get false positives so no need to be elaborate.